src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,11 @@
+package calculator
+
+import calculator.adder.Adder
+import calculator.separator.Separator
+
+class StringAddCalculator(private val adder: Adder) {
+    fun calculate(expression: String?): Int {
```

Comment:
```suggestion
    fun calculate(expression: String): Int {
```
nullable이 아닌 `non-nullable`한 값을 받아주는건 어떨까요?
사용자에게 값을 입력받을 때 validate를 해주면 비즈니스 로직이 조금 더 깔끔해질 것 같네요 :)